lib/modules/datasource/hex/schema.ts

```diff
@@ -0,0 +1,48 @@
+import { z } from 'zod';
+import { LooseArray } from '../../../util/schema-utils';
+import type { Release, ReleaseResult } from '../types';
+
+export const HexRelease = z
+  .object({
+    html_url: z.string().optional(),
+    meta: z
```

Comment:
`meta` is a confusing name to retain for this URL string

Reply:
this is coming from the json server result, so we can't change it here

Reply:
We are flattening it though. It makes sense as an object but not as a string

Reply:
@zharinov can this be left as meta.links.GitHub (optional) during the validation and only transformed later?